app/api/v1/endpoints/roles.py

```
@@ -32,14 +72,125 @@ async def list_roles(
 ):
     """Get role by id (tenant-scoped)."""
     role = await role_repo.get_by_id_and_tenant(role_id, tenant_id)
     if not role:
         raise HTTPException(status_code=404, detail="Role not found")
     return RoleResponse.model_validate(role)
+
+
+@router.put("/{role_id}", response_model=RoleResponse)
+@limit_writes
+async def update_role(
+    request: Request,
```

Ruff (0.15.0)

[warning] Unused function argument: `request`

(ARG001)